src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/DefaultComesLastCheck.java

```diff
@@ -47,31 +47,64 @@
  * <pre>
  * &lt;module name=&quot;DefaultComesLast&quot;/&gt;
  * </pre>
- * <p>
- * To configure the check for skipIfLastAndSharedWithCase:
+ * <p>Example:</p>
+ * <pre>
+ * switch (i) {
+ *   case 1:
+ *     break;
+ *   case 2:
+ *     break;
+ *   default:
+ *     break; // OK
```

Comment:
violation is on "default" token, please move ok comment on the line with "default"

Reply:
done